### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by logic-finder**
#### Summary
just little modification on [AudioParam.value document](https://developer.mozilla.org/en-US/docs/Web/API/AudioParam/value)
- graduated → gradiated
- add a space `//which is...` → `// which is...`

#### Motivation

> If any **gradiated** or ramped value changing methods have been called and the current time is within the time range over which the **graduated** change should occur, the value is updated based on the appropriate algorithm.

- According to the context of the sentence, it seems `gradiated` is correct.
- I think it is better to have a space in a comment.

#### Supporting details

none

#### Related issues

none

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/audioparam/value/index.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ the value of a parameter:
   {{domxref("AudioParam.setValueAtTime", "setValueAtTime()")}}, the `value`
   is changed to the value passed into `setValueAtTime()`.
 - If any gradiated or ramped value changing methods have been called and the current
```
Comment (wbamberg) [inline files/en-us/web/api/audioparam/value/index.md:94]:
```suggestion
- If any graduated or ramped value changing methods have been called and the current
```

File: files/en-us/web/api/audioparam/value/index.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ the value of a parameter:
...
-  time is within the time range over which the graduated change should occur, the value
+  time is within the time range over which the gradiated change should occur, the value
```
Comment (wbamberg) [inline files/en-us/web/api/audioparam/value/index.md:95]:
```suggestion
  time is within the time range over which the graduated change should occur, the value
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for this PR! I think we should change the other one, I'm not sure "gradiated" is a word, and if it is, it doesn't seem to be defined in the Web Audio spec.

**Comment by logic-finder** [comment]
!! I have thought that 'to graduate' means to get a diploma, so during translation I thought 'why this verb had been used here?', but I just realized it has another meaning. (I regarded 'gradiate' as a word related to 'gradient', so I thought this one was correct)

Thank you for giving me a chance to think about this.

p.s. What made me more surprised was that two words 'graduate' and 'gradient' have a same word origin: a Latin 'gradus', which means 'degree, step'. and it makes sense!

**Review by wbamberg — APPROVED** [review]
Ha, yes, it does have another meaning :). I wasn't sure if "gradiated" might also have some technical meaning that I was unaware of, but I can't find one.

Thanks for the fix @logic-finder !

**Pull request merged**